src/Operators/spectralelement.jl

```diff
@@ -404,25 +412,38 @@ end
 
 
 """
-    WeakDivergence()
-
-Computes the divergence of a vector field `v` weakly.
-
-This spolves the variational problem of finding `θ` such that
-
-    ⟨ϕ, J θ⟩ = - ⟨∂ϕ/∂ξ¹, J v¹⟩ + ⟨∂ϕ/∂ξ², J v²⟩
-
-for all `ϕ` (which arises by integration by parts).
-
-Discretely it is equivalent to
-
-    - J \\ (D₁' * W * J * v¹ + D₂' * W * J * v²)
-
+    wdiv = WeakDivergence()
+    wdiv.(u)
+
+Computes the "weak divergence" of a vector field `u`. This is defined as the
+scalar field ``\\theta`` such that for all ``\\phi``
+```math
+\\int_\\Omega \\phi \\theta \\, d \\Omega
```

Comment:
I would be careful with defining `\theta` here. Otherwise, don't just use another letter, since you already have `u`. I would just use a sample equation, say, a wave equation or a heat equation, where you have du/dt on the left hand side

Reply:
Then, if it's just to document the weak spatial operator, why do we need a LHS at all?

Reply:
> Since you define the weak derivative of u, I'd just use u and not introduce another letter

I was actually thinking of using theta for the weak derivative as well, since then it would be consistent

Reply:
Ok, sounds good. Yes, I see it now and it's better to be consistent

Reply:
Yes, that'd be good. Usually, in Finite Elements library that I've seen (e.g., libCEED and MFEM), it is customary to show at least one example that leads to one type of operator (i.e., just a L2 projection problem that leads to the Mass matrix; just an advection problem; and just a Laplace (Poisson) problem --- even just steady state. No need for the time derivative and linear solve, really). 
These 3 scenarios respectively lead to having the 3 different possible formulations: `interp-interp` (meaning, for an L2 projection, you end up with interpolation of your trial function `u` and test function `phi`); for a divergence only operator, `interp-grad` (meaning, you end up with interpolation of your trial function `u` and gradient of your test function `phi`); and the Laplacian leads to `grad-grad` (meaning, you end up with gradient of your trial function `u` and gradient of your test function `phi`)